### PR TITLE
Initial implementation for balance modal.

### PR DESCRIPTION
**Pull request by ajp8164**
When the total wallet balance is not equal to the available (spendable) balance then a button appears in the wallet home header.  The button text is the available (spendable) balance.

The total wallet balance is not equal to the available (spendable) balance when there are unconfirmed or locked funds.

Clicking the button presents four amounts and some explanation for each:

- Total balance
- Available balance
- Confirming amount
- Locked amount

Testing/fixing - this PR needs some UX work wrt. the collapsing wallet header and the size of the header when the button is displayed vs. not.

See https://github.com/bitpay/copay/issues/5194.

### CODE REVIEW

File: src/js/controllers/modals/walletBalance.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+angular.module('copayApp.controllers').controller('walletBalanceController', function($scope) {
+
+  $scope.close = function() {
+    $scope.walletBalanceModal.hide();
```
Comment (JDonadio) [inline src/js/controllers/modals/walletBalance.js]: Please, if the controller will not have more logic just use the next line on *Close* button in the HTML tag

> ng-click="walletBalanceModal.hide()"

And delete the controller

**Comment by ajp8164** [comment]
It may have more logic later.  I had thought about adding a "refresh" button to refresh balances.  We may also want to consider providing details behind confirming and locked amounts (like lists of tx and txp's).

But we can remove if for now and add the t back later as we need.

--
Andy Phillipson
Director of Engineering Services, BitPay, Inc.
844A CCA5 F277 30C0 E86F  9799 0CEE 502F 2484 0854

File: www/views/modals/wallet-balance.html [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+              </span>
+            </div>
+          </span>
+        </div>
+      </div>
+      <div class="wallet-balance__description" translate><b>Total</b> is the total amount of bitcoin stored in this wallet. It is the sum of Available, Confirming, and Locked amounts.</div>
+      <div class="wallet-balance__item item">
+        <i class="wallet-balance__icon icon ion-ios-checkmark-outline"></i>
+        <div class="wallet-balance__content no-border">
+          <div class="wallet-balance__title">Available</div>
```
Comment (gabrielbazan7) [inline www/views/modals/wallet-balance.html]: needs translate

Comment (ajp8164) [inline www/views/modals/wallet-balance.html]: Done.

**Comment by JDonadio** [comment]
ACK. Great job @ajp8164 !

**Comment by tonygallippi** [comment]
ACK.  works exactly as expected.  I approve

**Pull request merged**